website/routes/ui/people.py

```diff
@@ -157,11 +165,8 @@ def person(username):
     )
 
 
-@people.route("/<username>/edit/", methods=["GET", "POST"])
+@people.route("/people/<username>/edit/", methods=["GET", "POST"])
```

Comment:
The `people` route already has "/people" prepended by default (in `/website/__init__.py`), so you can remove it from urls in this file